mix.exs

```diff
@@ -27,7 +27,7 @@ defmodule UeberauthTwitter.Mixfile do
     [
      {:httpoison, "~> 0.7"},
      {:oauther, "~> 1.1"},
-     {:poison, "~> 1.3 or ~> 2.0"},
+     {:poison, "~> 1.3 or ~> 2.0 or ~> 3.0 or ~> 4.0"},
```

Comment:
@Virviil I'd rather we not continue to add versions to this for the foreseeable future, especially since this is something that can be handled in an individual project using the `override: true` option in mix:

```elixir
{:poison, "~> 4.0", override: true}
```

Let's either update this to `"~> 1.3 or > 2.0"` or make no change and add a comment to the README pointing folks to the `:override` option in Mix.